File: apps/android/app/src/main/java/com/rsr41/oracle/domain/model/CalendarType.kt
package com.rsr41.oracle.domain.model

enum class CalendarType(val apiValue: String, val displayName: String) {
    SOLAR("solar", "양력"),
    LUNAR("lunar", "음력");

    companion object {
        fun fromApiValue(value: String): CalendarType = entries.find { it.apiValue == value } ?: SOLAR
    }
}

File: apps/ios/OracleIOS/Domain/Entities/CalendarType.swift
import Foundation

/// 달력 타입 열거형
/// Android: domain/model/CalendarType.kt (lines 3-10)
enum CalendarType: String, Codable, CaseIterable, Identifiable {
    case solar = "solar"
    case lunar = "lunar"
    
    var id: String { rawValue }
    
    var displayName: String {
        switch self {
        case .solar: return L("common.solar")
        case .lunar: return L("common.lunar")
        }
    }
}
